templates/video_page.html

```diff
@@ -0,0 +1,69 @@
+{% extends 'base.html' %} {% block content %}
+<div class="row justify-content-center" style="text-align: center">
+  <h1>Enter video link below</h1>
+</div>
+<div
+  class="row justify-content-center"
+  style="text-align: center; padding: 0px"
+>
+  <p>
+    Keep in mind this tool is in early stages.
+  </p>
```

Comment:
Maybe later we should this off because I don't know if it sounds very professional.

Reply:
Everything else looks good tho imo